lms/templates/courseware/progress.html

```diff
@@ -78,6 +78,26 @@ <h4 class="hd hd-4 title">${certificate_data.title}</h4>
                 </div>
                 %endif
 
+                %if upgrade_available:
```

Comment:
I'd recommend making this into a separate template so that it can be dropped into other pages.